.github/workflows/0-welcome.yml

```
@@ -1,77 +1,75 @@
 name: Step 0, Welcome
 
-# This step listens for the learner pushing a commit to `main`.
-# This workflow updates from step 0 to step 1.
-
-# This will run every time we push a commit to `main`.
-# Reference: https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows
 on:
-  workflow_dispatch:
   push:
     branches:
       - main
 
 permissions:
   contents: write
-  pull-requests: write
+  actions: write
   issues: write
 
+env:
+  STEP_1_FILE: ".github/steps/1-resolve-duplicate-issues.md"
+
 jobs:
-  # Get the current step to only run the main job when the learner is on the same step.
-  get_current_step:
-    name: Check current step number
+  start_exercise:
+    if: |
+      !github.event.repository.is_template
+    name: Start Exercise
+    uses: skills/exercise-toolkit/.github/workflows/start-exercise.yml@v0.3.0
+    with:
+      exercise-title: "Connect the Dots"
+      intro-message: "Learn to navigate a GitHub repository effectively."
```

Slightly more engaging

```suggestion
      intro-message: "Let's learn to navigate a GitHub repository effectively!"
```